library_generation/test/resources/goldens/pr_description-golden.txt

```diff
@@ -0,0 +1,18 @@
+BEGIN_COMMIT_OVERRIDE
+BEGIN_NESTED_COMMIT
+Update repo-level parameter gapic_generator_version to 1.2.3
```

Comment:
Can we change this to something like `fix(deps): Update the Java code generator (gapic-generator-java) to 2.39.0` to keep it consistent with the existing Owlbot behavior? See https://github.com/googleapis/java-bigtable/pull/2191

Reply:
We need a mapping to do this, from parameter name to description.

What's the benefits of doing this? We don't have to generate the same message.

Reply:
> What's the benefits of doing this? I think we don't have to generate the same message as long as the meaning of this message is clear.

I agree. But I think the previous message makes more sense for handwritten library owners. Because
1. It includes a conventional commit prefix.
2. They don't need to know something like `repo-level parameter`, it is an implementation detail of hermetic build scripts. On high level, they just need to know the generator version is updated and the GAPIC layer is regenerated.

Reply:
Sure, this does make sense.

I need to create a mapping from parameter to commit message and I'll only add generator version and libraries_bom version for now. Other parameter will fallback to default message.

Reply:
Done.